client/src/components/refreshrs/Refreshr.js

```diff
@@ -30,33 +30,62 @@ const styles = (theme) => ({
   },
   subheaders: {
     marginTop: '.5rem'
+  },
+  answerFields: {
+    height: '100%',
+    width: '100%',
+    maxWidth: '160px',
+    maxHeight: '28px'
   }
 });
 
 function Refreshr(props) {
-  const [ studentName, addStudentName ] = useState("");
-  const [ reviewText, setReviewText ] = useState("This is a short section of text that describes or reminds about one part of the topic. It is not intended to be a complete review of the material, just a reminder of the most important parts. This section is limited to 512 characters.");
-  const [ questionText, setQuestionText ] = useState("This is a multiple choice question related to the topic. Again the goal is not to be exhaustive, just to call the material to mind. A good question will not simply call for the recollection of information, but will require the application of knowledge.");
-  const [ a1, setA1 ] = useState(false);
-  const [ a2, setA2 ] = useState(false);
-  const [ a3, setA3 ] = useState(false);
-  const [ a4, setA4 ] = useState(false);
-  
-  console.log('"state"', {studentName, reviewText, questionText, a1, a2, a3, a4}, 'Props', props)
+  const [refreshrName, addRefreshrName] = useState('');
+  const [reviewText, setReviewText] = useState(
+    'This is a short section of text that describes or reminds about one part of the topic. It is not intended to be a complete review of the material, just a reminder of the most important parts. This section is limited to 512 characters.'
+  );
+  const [questionText, setQuestionText] = useState(
+    'This is a multiple choice question related to the topic. Again the goal is not to be exhaustive, just to call the material to mind. A good question will not simply call for the recollection of information, but will require the application of knowledge.'
+  );
+  const [a1, setA1] = useState(false);
+  const [a2, setA2] = useState(false);
+  const [a3, setA3] = useState(false);
+  const [a4, setA4] = useState(false);
+  const [a1Text, setA1Text] = useState('');
+  const [a2Text, setA2Text] = useState('');
+  const [a3Text, setA3Text] = useState('');
+  const [a4Text, setA4Text] = useState('');
+  const [questionObject, setQuestionObject] = useState({
+    refreshrName,
```

Comment:
Nice use of hooks! We can probably even refactor later to combine some of these into one object

Reply:
That's a really good idea.